### DISCUSSION TITLE
Include __name__ = 'main' to run plzCrawler?

### DISCUSSION THREAD

**Discussion by Sammeeey** [Sammeeey/plzCrawler#9] [Ideas]
Does it make sense to include a
```
if __name__ = 'main':
```
block in the [plzCrawler.py file](https://github.com/Sammeeey/plzCrawler/blob/5ad7f62800459fa8c3a01e2389d745ded169353d/plzCrawler_en/plzCrawler.py)?

I've just read the [[What does if __name__ == "__main__": do?](https://stackoverflow.com/questions/419163/what-does-if-name-main-do)](https://stackoverflow.com/a/419185) thread on stackoverflow.
And the approach sounds like something which might make sense for the plzCrawler tool.

The respective if-block may include the content from [plzMaker.py](https://github.com/Sammeeey/plzCrawler/blob/5ad7f62800459fa8c3a01e2389d745ded169353d/plzCrawler_en/plzMaker.py).

Currently the only use of the plzMaker.py file is to run the plzCrawler.py file and to trigger its respective classes/functions.
As far as I understand the functionality of the `if __name__ = 'main':` block, this approach could make the additional plzMaker.py file obsolte and simplify the tool.

What do you think?

**Comment by mikeysan** [comment]
From my understanding of the "if __name__ = main", I would agree with your analysis. I haven't looked at either file yet but that is the function of this code block. 

I will add any updates, if necessary, once I have looked at the individual files.

**Reply by Sammeeey** [reply]
Great, thank you for your feedback @mikeysan ✌
